docs/docs/user-guide/access-control.md

```diff
@@ -0,0 +1,16 @@
+# Overview of Access Control in Mathesar
+
+To manage access to data, Mathesar utilizes PostgreSQL's powerful role-based permissions system. Each user accesses a database through a specific PostgreSQL role, and the user's access is determined by the role's privileges within PostgreSQL.
```

Comment:
I'm struggling a bit with the second sentence. I'm uncertain if this is even better, but how about something like:


> Mathesar manages data access using PostgreSQL's powerful role-based permissions system. Users interact with the database through designated PostgreSQL roles, with their access determined by the privileges assigned to those roles.

Reply:
Sounds good. Can you implement this?